docs/_data/paths/rating-proposals_realtime.yaml

```diff
@@ -1,12 +1,87 @@
+get: &get
+  operationId: getRealTimeProposalStatus
+  summary: Obtain the status of the Ratings Provider's Real-Time proposals
+  description: |
+
+    Used to obtain the status of real-time ratings proposals. The response is
+    implicitly restricted to the requesting Ratings Provider's obligation.
+    Accordingly, the caller can use this endpoint to check the state of their
+    proposal submission.
+
+    Note that the same status object is returned for each
+    `postRealTimeProposal`, so this endpoint may seem redundant. However,
+    an anticipated use case for this `GET` endpoint is to support supervisor
+    processes that are setup by the client to independently ensure the provider's process for
+    rating submission if functioning properly. For example, the Rating Provider
+    might have one program responsible for producing and submitting Ratings
+    Forecasts via `postRealTimeProposal`, while having a separate
+    monitoring job that checks this endpoint regularly.
+
+    Clients SHOULD perform Conditional `GET` using the `If-None-Match` header
```

Comment:
I think we should have put this paragraph in the GET limits snapshot instead of the status, or at least put it in both places.

Reply:
Done, and I think this cleans up the PR